### DISCUSSION TITLE
Trouble loading DIC map

### DISCUSSION THREAD

**Discussion by James-Sharples** [MechMicroMan/DefDAP#99] [Q&A]
Hi all, I am following the example notebook but cannot set the location of my DIC images. here is the code.
# set the path of the pattern image, this is relative to the location of the DIC data file
#dicMap.setPatternPath("0000-1.tif", 1)
dicMap.set_pattern("stitched_threshold_coarse.bmp") 

Any help is much appreciated,
James S

**Comment by James-Sharples** [comment]
Here is the error message I am getting.
AttributeError                            Traceback (most recent call last)
Input In [9], in <cell line: 2>()
      1 #dicMap.setPatternPath("0000-1.tif", 1)
----> 2 dicMap.set_pattern("stitched_threshold_coarse.bmp",1)

AttributeError: 'Map' object has no attribute 'set_pattern'

**Reply by JQFonseca** [reply]
I think the method is now called dicMap.setPatternPath, is that right? This is commented out in the notebook and a bit confusing. Could do with fixing. I would be happy to do it.
